tests/Http/PostTest.php

```diff
@@ -585,6 +607,28 @@ public function testPostsIndexAsAdmin()
         ]);
     }
 
+    /**
+     * Test for retrieving all posts as admin
+     * Admins should see hidden posts.
+     *
+     * GET /api/v3/posts
+     * @return void
+     */
+    public function testPostsIndexAsAdminHiddenPosts()
+    {
+        // Admins should see all posts.
+        factory(Post::class, 'accepted', 10)->create();
+
+        $hiddenPost = factory(Post::class, 'accepted')->create();
+        $hiddenPost->tag('Hide In Gallery');
+        $hiddenPost->save();
+
+        $response = $this->withAdminAccessToken()->getJson('api/v3/posts');
+
```

Comment:
@DFurnes it's weird - when I do `dd($response->decodeResponseJson());` here, the response doesn't include a `tags` array for each post like we expect - could this be part of the issue?

Reply:
Hmm, that's interesting. It shoooould. 🤔

Reply:
hmm so when I comment out `$hiddenPost->tag('Hide In Gallery);` and saving it, then the `tags` array is returned. It seems like it is something with those lines that is making it upset!